bound/kt/src/test/kotlin/web5/sdk/TestVectorsTest.kt

```diff
@@ -0,0 +1,104 @@
+package web5.sdk
+
+import com.fasterxml.jackson.core.type.TypeReference
+import org.junit.jupiter.api.Test
+import java.io.File
+import org.junit.jupiter.api.Assertions.*
+import web5.sdk.crypto.keys.Jwk
+import web5.sdk.crypto.signers.Ed25519Signer
+import web5.sdk.crypto.verifiers.Ed25519Verifier
+
+class TestVectors<I, O>(
+    val description: String,
+    val vectors: List<TestVector<I, O>>
+)
+
+class TestVector<I, O>(
+    val description: String,
+    val input: I,
+    val output: O?,
+    val errors: Boolean? = false,
+)
+
+class Web5TestVectorsCryptoEd25519 {
+
+    data class SignTestInput(
+        val data: String,
+        val key: TestVectorJwk,
+    )
+
+    data class TestVectorJwk(
```

Comment:
I know we're not testing the JWK in this test, but ideally we could use the real `Jwk`, but to your point on the other PR, we may need to add #297 first?

Reply:
yup after 297 is in we can use it